### PR TITLE
taints and labels machinepool test

### PR DESCRIPTION
**Pull request by amalykhi**
Automation of Polarion TC https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-64905
and
https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-64904

The tests succeeded on local
Ran 4 of 8 Specs in 39.001 seconds
SUCCESS! -- 4 Passed | 0 Failed | 0 Pending | 4 Skipped

### CODE REVIEW

**Comment by amalykhi** [comment]
Please, review PR @shellymiron @tshefi @xueli181114 @yasun1

File: tests/e2e/machine_pool_test.go [hunk 1]
```diff
@@ -107,5 +107,115 @@ var _ = Describe("TF Test", func() {
...
+				}
+
+				err := mpService.Create(MachinePoolArgs)
+				Expect(err).ToNot(HaveOccurred())
+				By("Verify the parameters of the created machinepool")
+				mpResponseBody, err := cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, name)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(mpResponseBody.Autoscaling().MinReplicas()).To(Equal(minReplicas))
+				Expect(mpResponseBody.Autoscaling().MaxReplicas()).To(Equal(maxReplicas))
+				By("Change the number of replicas of the machinepool")
```
Comment (shellymiron) [inline tests/e2e/machine_pool_test.go:135]: can we do spaces between `By()` to `By()`? just for visibility of the steps taken

**Comment by shellymiron** [comment]
besides my previous comment, /lgtm

**Pull request merged**